### ISSUE TITLE
Web pages fail HTML5 markup validation

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3102]
**HC Assessment label: Moderate**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest,
Feedback on our Accessibility,
Thank you for submitting an expression of interest,
Thank you for your feedback
URL: https://forms-staging.cdssandbox.xyz/en/id/134
https://forms-formulaires.alpha.canada.ca/en/id/clq1bz2xl005ax382blj94w82
https://forms-staging.cdssandbox.xyz/en/id/134/confirmation
https://forms-formulaires.alpha.canada.ca/id/clq1bz2xl005ax382blj94w82/confirmation
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
The pages fail HTML5 markup validation. The WC3 Markup Validation tool reports markup errors. Generated page source is used to validate the markup using the WC3 Markup Validation Service (https://validator.w3.org/#validate_by_input).

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Access the Web Developer Tools using the F12 key
3. Go to "Elements" tab
4. Right click on the element
5. Then, click "Edit as HTML" from the context menu
6. Open the W3C validator and go the “Validate by direct input” tab
7. Add the doctype “<!doctype html>” in the text input field in the W3C validator
8. Then, copy the generated HTML source from the Web Developer tools and paste in the W3C validator
9. Press the "Check" button

Expected result: All the markup errors should be addressed on the pages.

**Comment by srtalbot** [comment]
@thiessenp-cds - were you able to diagnos this issue or fix it? 

(Not asking you to pick it up, but want to know if we should close it)

**Comment by srtalbot** [comment]
We won't be fixing this, as our system uses JSON not HTML mark-up, which is causes the accessibility testing tools to fail but do not actually produce any accessibility issues. 

cc: @mcman12 for an example of a client flagging JSON as an issue.
